package.json

```
@@ -32,6 +32,8 @@
     "prepack": "./scripts/prepack.sh",
     "setup": "yarn install",
     "test": "jest",
+    "test:debug": "node --inspect-brk node_modules/.bin/jest --runInBand",
+    "test:debug:windows": "node --inspect-brk ./node_modules/jest/bin/jest.js --runInBand",
```

I feel like we should minimize the number of scripts we have, so is there a way that we can look for the location of `jest`? Perhaps we can extract this to a script in `scripts/` and say something like:

``` sh
#!/bin/sh

exec node --inspect-brk $(yarn bin jest) --runInBand
```

That way we only need one script.

---

nice one! what if we just put that straight in the package.json script for `test:debug` ?

```suggestion
    "test:debug": "node --inspect-brk $(yarn bin jest) --runInBand",
```

---

This wouldn't work on Windows though. `cmd` doesn't support the `$( .. )` syntax.

I'd recommend using [`yarn run`](https://yarnpkg.com/cli/run#options), it supports the `--inspect-brk` command.

e.g.

```
"test:debug": "yarn run --inspect-brk jest --runInBand",
```

---

@Gudahtt ooooooo nice catch!